cmd/config/config.go

```diff
@@ -305,3 +319,33 @@ func writeAlphabetizedFlags(sb *strings.Builder) {
 		sb.WriteString(fmt.Sprintf("- `%s`: %s\n", flag, cliflags.AllFlagsHelp()[flag]))
 	}
 }
+
+func outputSetAction(newFields []string) (string, error) {
```

Comment:
These functions create JSON to pass into the output function since it expects an API response.